packages/rpc-transformers/src/response-transformer.ts

```diff
@@ -6,14 +8,19 @@ export type ResponseTransformerConfig<TApi> = Readonly<{
     allowedNumericKeyPaths?: AllowedNumericKeypaths<TApi>;
 }>;
 
+type JsonRpcResponse = { error: Parameters<typeof getSolanaErrorFromJsonRpcError>[0] } | { result: unknown };
+
 export function getDefaultResponseTransformerForSolanaRpc<TApi>(config?: ResponseTransformerConfig<TApi>) {
```

Comment:
Yeah, this one. Maybe needs a return type annotation?

Reply:
Added the return type as `NonNullable<RpcApiConfig['responseTransformer']>`. I also needed to tweak the `rpc-spec` to make `methodName` optional on the response transformer, matching this implementation, which I think makes sense anyway.